src/main/java/uk/gov/hmcts/reform/lrdapi/controllers/advice/InvalidRequestException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.lrdapi.controllers.advice;
+
+public class InvalidRequestException extends RuntimeException {
+
+    public InvalidRequestException(String message) {
+        super(message);
```

Comment:
why are we creating so many exception classes? I think one should fine LrdApiException

Reply:
This one is used for 400 error code scenario and LrdApiException is used for internal server error scenario's